webpack/stories/docs/addingNewComponent.md

```diff
@@ -0,0 +1,98 @@
+
+An example of such stories is [BreadcrumbBar.stories.js](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.stories.js).
+
+On top of the basic storybook Foreman uses [knobs addon](https://github.com/storybooks/storybook/tree/master/addons/knobs#available-knobs) that enables switching values of component's properties and [actions addon](https://github.com/storybooks/storybook/tree/master/addons/actions#getting-started) for logging of callbacks. It's worth checking their docs as the usage is simple and they can provide nice user experience.
+
+
+### Running storybook
+
+Starting the storybook server is as easy as `npm run storybook`. By default the server is started on `localhost:6006`.
+
+If some extra configuration is needed, you can start the storybook's executable directly. For example:
+```bash
+# Starting storybook on https with Katello certs
+./node_modules/.bin/start-storybook \
+  --port 6006 \
+  --host $(hostname) \
+  --https \
+  --ssl-key /etc/pki/katello/private/katello-apache.key \
+  --ssl-cert /etc/pki/katello/certs/katello-apache.crt \
+  --ssl-ca /etc/pki/katello/certs/katello-default-ca.crt
+```
+
+## Testing
+
+Tests must be placed in `__tests__` subfolder of the main component's folder. Tests for the component, reducer and actions must have their own files named `<TESTED_PIECE>.test.js`:
+
+```
+╰─ __tests__
+    ├─ <COMPONENT_NAME>Actions.test.js
+    ├─ <COMPONENT_NAME>Reducer.test.js
+    ├─ <COMPONENT_NAME>.test.js
+    ├─ integration.test.js
+    ╰─ __snapshots__
+        ├── # All snapshot files (created automaically, updated with `npm test -- -u`)
+```
+
+### Testing components
+
+In most cases (when the component doesn't provide any user interaction callbacks) it's enough to test how the component is rendered with various supported properties. Foreman uses [enzyme](https://github.com/airbnb/enzyme) for that.
+
+There are 3 ways how a component can be rendered in enzyme:
+  - **shallow** - render subcomponents as names, **preferred alternative**, more details in [docs](https://github.com/airbnb/enzyme/blob/master/docs/api/shallow.md)
+  - **mount** - full rendering API, useful when you need to interact with dom, more details in [docs](https://github.com/airbnb/enzyme/blob/master/docs/api/mount.md)
+  - **render** - static rendering, more details in [docs](https://github.com/airbnb/enzyme/blob/master/docs/api/render.md)
+
+### Running the tests
+
+All tests can be executed with `npm test`.
+
+If you want to run only a single test use `jest` directly: `./node_modules/.bin/jest <path/to/some.test.js>`
+This is useful especially for debugging because it doesn't hide console output.
+
+Linter (code style checking) can be executed with `npm run lint`.
```

Comment:
should we add here - 
a. a simple way to fix lint errors is with `--fix` 
b. a recommendation to use code formatter
?